expectise/expect.py

```diff
@@ -285,5 +286,9 @@ def tear_down(cls) -> None:
         for key in temporary_mocks:
             cls.method_h.pop(key)
 
+        # In case an exception was raised during the test, raising it again
+        if execution_error:
+            raise execution_error
```

Comment:
at which point was it swallowed previously? inherent behaviour to `__exit__` ?

Reply:
yeah, the exit method catches errors.